<launch>

    <arg name="events_dat_file"    default="$(find reconstruction_3d_event_camera_pkg)/data/thetacalc1hzlowsen_td.dat"/>
    <arg name="events_bag_file"    default="$(find reconstruction_3d_event_camera_pkg)/results/bag_file.bag"/>

    <!-- Create ROS Bag from the .dat file-->
    <node pkg="reconstruction_3d_event_camera_pkg"    type="rosbag_from_dat.py"   name="publish_events_node" output="screen">
        <param name="events_dat_file"   type="str"  value="$(arg events_dat_file)"/>
        <param name="events_bag_file"   type="str"  value="$(arg events_bag_file)"/>
    </node>

</launch>